api/src/main/java/org/apache/iceberg/actions/ExpireSnapshots.java

```diff
@@ -109,5 +109,8 @@ interface Result {
 
     /** Returns the number of deleted manifest lists. */
     long deletedManifestListsCount();
+
+    /** Returns the number of deleted statistics files. */
+    long deletedStatisticsFilesCount();
```

Comment:
in order to not introduce breaking API changes, this should be a default method and return 0, similar to how we did it in #6801